moped-editor/src/views/projects/projectsListView/ProjectsListViewMap.js

```diff
@@ -1,11 +1,98 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { useQuery } from "@apollo/client";
 import ProjectsMap from "./components/ProjectsMap";
+import Alert from "@mui/material/Alert";
+import { GET_PROJECTS_GEOGRAPHIES } from "src/queries/project";
+import { styleMapping } from "../projectView/ProjectStatusBadge";
 
-const ProjectsListViewMap = () => {
+const ProjectsListViewMap = ({
+  mapQuery,
+  fetchPolicy,
+  setIsMapDataLoading,
+}) => {
   /* Store map instance to call Mapbox GL methods where needed */
   const mapRef = React.useRef();
 
-  return <ProjectsMap ref={mapRef} />;
+  const { data: projectMapViewData } = useQuery(mapQuery, {
```

Comment:
All of the data fetching is very rough draft but it gets features on the map. I'm seeing UI freezing/locking when switching on the map with no filters applied so we'll be revisiting.

Reply:
this might be overkill, and it's not going to address rendering issues, but you could allow the fetching and data munging to happen even when the map is not rendered, so that it's ready to go when some toggles the map tab 🤷

Reply:
i'm going to keep this mind - i think that it would be a good next step if that initial view feels too long!